frontend/src/app/forms/forms.page.ts

```diff
@@ -64,6 +71,8 @@ export class FormsPage implements OnInit, OnDestroy {
   */
   private subscriptions: Subscription[] = [];
 
+  hideButton = false;
```

Comment:
Remove this property. Haven't used anywhere.

Reply:
done